mirror_builder/sources.py

```diff
@@ -64,6 +91,17 @@ def _patch_source(ctx, source_root_dir):
 
 
 def prepare_source(ctx, req):
+    logger.info('preparing source for %s', req)
```

Comment:
Maybe we should move the patching step here, too?

Reply:
Makes sense, done